### PR TITLE
[1LP][RFR] Some more tags test

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Added more test to cover tag visibility, added TaggableCollection class to give an oportunity to select multiple items for tagging
Shriver: updated pytest line to include more tests
{{pytest: -k 'test_tag_host_after_provider_delete or test_multiple_cloud_volumes_tag_edit or test_tagvis_provision_fields or test_restricted_catalog_items_select_for_catalog_bundle' -vvv}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ae6e04d2aea23b453d4ccde3d0f8ca42de81ce01:

`cfme/tests/services/test_catalog_item.py`:
- :bangbang: Line 157:5: **E125** *continuation line with same indent as next logical line*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cf44f14a32eebd01b8a1beec855522acae1ff039:

`cfme/tests/cloud/test_providers.py`:
- :heavy_exclamation_mark: Line 630:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/cloud/instance/image.py [hunk 1]
```diff
@@ -160,6 +160,7 @@ def all(self):
 
 
 @navigator.register(ImageCollection, 'All')
+@navigator.register(Image, 'All')
```
Comment (mshriver) [inline cfme/cloud/instance/image.py]: :+1: :+1: Thanks @rlbabyuk

File: cfme/storage/volume.py [hunk 2]
```diff
@@ -363,6 +364,31 @@ def delete(self, *volumes):
         else:
             raise VolumeNotFoundError('No Cloud Volume for Deletion')
 
+    def all(self):
```
Comment (mshriver) [inline cfme/storage/volume.py]: :+1: thanks for adding this!

**Pull request merged**